cluster/aws/util.sh

```diff
@@ -604,44 +599,25 @@ function kube-up {
 
   echo "Kubernetes cluster created."
 
-  local kube_cert="kubecfg.crt"
-  local kube_key="kubecfg.key"
-  local ca_cert="kubernetes.ca.crt"
   # TODO use token instead of kube_auth
-  local kube_auth="kubernetes_auth"
+  export KUBECONFIG="${KUBECONFIG:-"${HOME}/.kube/.kubeconfig"}"
```

Comment:
This code *works* but a nitpick of mine is nested double quotes which always look kind of ugly...

It turns out you can drop either pair of double quotes and it will still work exactly the same... In this particular case, I think I'd advise dropping the inner quotes since you have outer quotes everywhere else.

```shell
export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/.kubeconfig}"
```